src/pages/iou/request/step/IOURequestStepScan/index.native.tsx

```diff
@@ -193,41 +172,43 @@ function IOURequestStepScan({
         }
 
         // If the transaction was created from the global create, the person needs to select participants, so take them there.
-        if (isFromGlobalCreate && iouType !== CONST.IOU.TYPE.TRACK_EXPENSE) {
+        if (transaction?.isFromGlobalCreate && iouType !== CONST.IOU.TYPE.TRACK_EXPENSE) {
             Navigation.navigate(ROUTES.MONEY_REQUEST_STEP_PARTICIPANTS.getRoute(iouType, transactionID, reportID));
             return;
         }
 
         // If the transaction was created from the + menu from the composer inside of a chat, the participants can automatically
         // be added to the transaction (taken from the chat report participants) and then the person is taken to the confirmation step.
+
         IOU.setMoneyRequestParticipantsFromReport(transactionID, report);
```

Comment:
@codinggeek2023 Since the `report` we pass is an Onyx value, it can be `null`. You can just update [setMoneyRequestParticipantsFromReport](https://github.com/Expensify/App/blob/main/src/libs/actions/IOU.ts#L5051) function to accept `OnyxEntry<OnyxTypes.Report>` instead of just `OnyxTypes.Report`, it should help
```
function setMoneyRequestParticipantsFromReport(transactionID: string, report: OnyxEntry<OnyxTypes.Report>) {
```

Reply:
@codinggeek2023 As suggested by @VickyStash, just change the signature of the `setMoneyRequestParticipantsFromReport` function to allow nullable value:

```diff
diff --git a/src/libs/actions/IOU.ts b/src/libs/actions/IOU.ts
index 1ee6665cef..ba10242654 100644
--- a/src/libs/actions/IOU.ts
+++ b/src/libs/actions/IOU.ts
@@ -5048,9 +5048,9 @@ function replaceReceipt(transactionID: string, file: File, source: string) {
  * @param transactionID of the transaction to set the participants of
  * @param report attached to the transaction
  */
-function setMoneyRequestParticipantsFromReport(transactionID: string, report: OnyxTypes.Report) {
+function setMoneyRequestParticipantsFromReport(transactionID: string, report: OnyxEntry<OnyxTypes.Report>) {
     // If the report is iou or expense report, we should get the chat report to set participant for request money
-    const chatReport = ReportUtils.isMoneyRequestReport(report) ? ReportUtils.getReport(report.chatReportID) : report;
+    const chatReport = ReportUtils.isMoneyRequestReport(report) ? ReportUtils.getReport(report?.chatReportID) : report;
     const currentUserAccountID = currentUserPersonalDetails.accountID;
     const shouldAddAsReport = !isEmptyObject(chatReport) && ReportUtils.isSelfDM(chatReport);
     const participants: Participant[] =

```